docs/ldml/tr35-info.md

```diff
@@ -807,6 +815,8 @@ An implementation need not use rationals directly for conversion; it could use d
 <!ATTLIST convertUnit factor CDATA #IMPLIED >
 
 <!ATTLIST convertUnit offset CDATA #IMPLIED >
+
+<!ATTLIST convertUnit description CDATA #IMPLIED >
```

Comment:
Did you mean systems instead of description here?

Reply:
I meant the description attribute. The goal here is to add the  <!ATTLIST XXX description ...> lines that were missing, for the applicable XXX's

Reply:
Where is the "systems" attribute though?  I don't see any line that says, as stated in the ticket,

```
<!ATTLIST convertUnit systems NMTOKENS #IMPLIED >
<!--@MATCH:set/literal/ussystem, uksystem, metric, si, other -->
```